tests/containers/docker_image.pm

```diff
@@ -23,19 +24,21 @@ use Mojo::Base qw(consoletest);
 use testapi;
 use utils;
 use containers::common;
+use containers::utils;
 use containers::container_images;
 use containers::urls 'get_suse_container_urls';
 use version_utils qw(get_os_release check_os_release);
 
 sub run {
+    my ($self) = @_;
     my ($image_names, $stable_names) = get_suse_container_urls();
     my ($running_version, $sp, $host_distri) = get_os_release;
     my $runtime = "docker";
 
     install_docker_when_needed($host_distri);
     allow_selected_insecure_registries(runtime => $runtime);
     scc_apply_docker_image_credentials() if (get_var('SCC_DOCKER_IMAGE'));
-
+    check_docker_firewall()              if (check_runtime_version(get_docker_version(), ">=20.10") && !($host_distri ne 'centos' || $host_distri ne 'ubuntu'));
```

Comment:
`!($host_distri ne 'centos' || $host_distri ne 'ubuntu'))` is always `true`. Did you mean

`!($host_distri eq 'centos' || $host_distri eq 'ubuntu'))`? AFAICT checking for openSUSE/SLE specifically makes more sense

Reply:
It makes no sense to run this on `docker.pm` as well as here. It's enough in `docker.pm`.

Reply:
I also consider routines `check_runtime_version` and `get_docker_version` as over-engineering.
One `script_output` condition should be enough.

Reply:
> It makes no sense to run this on `docker.pm` as well as here. It's enough in `docker.pm`.

i am not sure what do you mean here. i added initially to the docker.pm. But we have scheduled tests that run only the `containers_sle_image_on_sle_host` with each container build and do not have the docker.pm.

Reply:
> I also consider routines `check_runtime_version` and `get_docker_version` as over-engineering.
> One `script_output` condition should be enough.

Maybe a bit. but i believe they make sense. Especially the `check_runtime_version`.

Reply:
> `!($host_distri ne 'centos' || $host_distri ne 'ubuntu'))` is always `true`. Did you mean
> 
> `!($host_distri eq 'centos' || $host_distri eq 'ubuntu'))`? AFAICT checking for openSUSE/SLE specifically makes more sense

fixed

Reply:
> > It makes no sense to run this on `docker.pm` as well as here. It's enough in `docker.pm`.
> 
> i am not sure what do you mean here. i added initially to the docker.pm. But we have scheduled tests that run only the `containers_sle_image_on_sle_host` with each container build and do not have the docker.pm.

True, for the Latest we have those. But we should then also have test sute for testing the runtime (e.g. `docker.pm`, `podman.pm`, `buildah.pm`). @jlausuch wdyt?